src/MessageWriter.ts

```diff
@@ -203,6 +213,15 @@ export class MessageWriter {
           writer.sequenceLength(arrayLength);
```

Comment:
Just wondering, it seems like these CdrWriter `writer.*array` methods have a `writeLength` parameter... why is the MessageWriter not using that but instead doing its own `sequenceLength` here?

Reply:
Not sure, maybe that's something @jhurliman can answer?

Reply:
I forget if there is an intentional reason or not :-/

Reply:
You should definitely be able to use that parameter, it could've been added after this was implemented?

Reply:
Not sure if it makes sense to change it in this PR. I would leave it for later